supplementary_style_guide/style_guidelines/formatting.adoc

```diff
@@ -80,7 +80,7 @@ Before you begin to customize the installer, download the Red{nbsp}Hat-provided
 [product-names-versions-ref]]
 == Product names and version references
 
-Whenever you refer to the name of your product in full, or in its abbreviated form, or when you refer to the major and minor version of your product, avoid using hard-coded references and use attributes instead.
+Use attributes instead of hard-coded references when you refer to the name of your product in full, or in its abbreviated form, or when you refer to the major and minor version of your product.
```

Comment:
```suggestion
Use attributes instead of hard-coded references when you refer to the name of your product in full, to its abbreviated form, or to its major or minor version.
```

Reply:
Much better. Thank you. Updated.